src/etl/construct_etl.py

```diff
@@ -1,27 +1,24 @@
+'''Construct ETL'''
+
 import logging
 import multiprocessing
 import uuid
-from test import TestObject
-import time
 from etl.helpers import ResourceDescriptorHelper
-from common import ContextInfo
 from etl import ETL
-from etl.helpers import ETLHelper
+from etl.helpers import ETLHelper, TextProcessingHelper
```

Comment:
should this be its own line to match the rest of the import break out that you've nicely done?